src/_ert/forward_model_runner/client.py

```diff
@@ -55,84 +51,83 @@ def __init__(
         cert: Optional[Union[str, bytes]] = None,
         max_retries: Optional[int] = None,
         timeout_multiplier: Optional[int] = None,
+        dealer_name: str | None = None,
```

Comment:
Too keep it the same style as the other parameters
```suggestion
        dealer_name: Optional[str] = None,
```

Reply:
I wanted to be "modern", but will make it the same then :)

Reply:
After Eivind's typing PR, maybe we should have your original version after all. That way it wont be a rebase conflict right before you try merging it 😆

Reply:
Yes, it already back as Optional got removed.